src/app/about/about.component.ts

```diff
@@ -27,6 +27,8 @@ export class AboutComponent implements OnInit {
     new Sponsor('terra.png', new URL('https://terra.bio')),
```

Comment:
Outside the scope of this PR, but happened to notice on line 25 -- @denis-yuen , now that we removed the launch with DNAstack button, should we remove them as an integration partner as well?

Reply:
I'd say leave it. They're not a "launch-with" partner in the sense of having a button but they're an integration partner in the sense that workbench uses our API and advertises us.